config.json

```diff
@@ -124,6 +124,22 @@
         ],
         "status": "wip"
       },
+      {
+        "uuid": "0e7ecc5e-b4d6-4ead-bfaa-a2ebdddcff47",
+        "name": "Stage Heralding",
+        "slug": "stage-heralding",
+        "concepts": [
+          "regex"
+        ],
+        "prerequisites": [
+          "strings",
+          "conditionals",
+          "nothingness",
+          "symbols",
+          "getindex"
```

Comment:
This is something this exercise could also introduce. The idea of "you can use `foo[bat]` to access more objects than just dicts/vectors" probably doesn't require its own concept exercise.